enhance upstream Makefile to install gmplayer manpages

the debian package used to install gmplayer symlinks with custom ln
calls. It is way more convenient to implement this in the upstream
Makefile.

This patch should be ready to be applied upstream.

diff --git a/Makefile b/Makefile
index 263f3cd..f285283 100644
--- a/Makefile
+++ b/Makefile
@@ -899,6 +899,7 @@ install-gui: install-mplayer
 
 install-mencoder-man: $(foreach lang,$(MAN_LANGS),install-mencoder-man-$(lang))
 install-mplayer-man:  $(foreach lang,$(MAN_LANGS),install-mplayer-man-$(lang))
+install-mplayer-gui-man: $(foreach lang,$(MAN_LANGS),install-mplayer-gui-man-$(lang))
 
 install-mencoder-man-en: install-mplayer-man-en
 	cd $(MANDIR)/man1 && ln -sf mplayer.1 mencoder.1
@@ -918,8 +919,17 @@ install-mplayer-man-$(lang):
 	$(INSTALL) -m 644 DOCS/man/$(lang)/mplayer.1 $(MANDIR)/$(lang)/man1/
 endef
 
+define MPLAYER_GUI_MAN_RULE
+install-mplayer-gui-man-$(lang): install-mplayer-man-$(lang)
+	cd $(MANDIR)/$(lang)/man1/ && ln -sf mplayer.1 gmplayer.1
+endef
+
+install-mplayer-gui-man-en: install-mplayer-man-en
+	cd $(MANDIR)/man1/ && ln -sf mplayer.1 gmplayer.1
+
 $(foreach lang,$(filter-out en,$(MAN_LANG_ALL)),$(eval $(MENCODER_MAN_RULE)))
 $(foreach lang,$(filter-out en,$(MAN_LANG_ALL)),$(eval $(MPLAYER_MAN_RULE)))
+$(foreach lang,$(filter-out en,$(MAN_LANG_ALL)),$(eval $(MPLAYER_GUI_MAN_RULE)))
 
 uninstall:
 	rm -f $(BINDIR)/mplayer$(EXESUF) $(BINDIR)/gmplayer$(EXESUF)
